mailpoet/RoboFile.php

```diff
@@ -1126,6 +1126,9 @@ public function releasePublish($version = null) {
       ->addCode(function () {
         return $this->releaseDownloadZip();
       })
+      ->addCode(function () use ($version) {
+        return $this->releaseVerifyDownloadedZip();
```

Comment:
Probably a typo, but `$version` is not passed into the method. The same in the premium plugin,

```suggestion
        return $this->releaseVerifyDownloadedZip($version);
```